tests/test_ace_fitting.py

```diff
@@ -86,25 +78,39 @@ def test_ace_fit(request, tmp_path, monkeypatch, run_dir='run_dir'):
     (tmp_path / run_dir).mkdir()
 
     fit_config_file = os.path.join(os.path.dirname(request.fspath), 'assets', 'B_DFT_data.xyz')
-    params = { 'r0' : 1.8, 'body_order': 3, 'degree': 6 }
+    # Only mandatory params
+    params = {
+        "data": {},
+        "basis": {
+            "rpi": {"type": "rpi", "species": ["B"], "N": 3, "maxdeg": 6},
+            "pair": {"type": "pair", "species": ["B"], "maxdeg": 4}},
+        "solver": {"solver": "lsqr"}}
 
     # removed from fit, maybe should do here, if we actually want to test the resulting potential?
     # database_modify_mod='wfl.fit.modify_database.gap_rss_set_config_sigmas_from_convex_hull',
 
     t0 = time.time()
-    ACE = fit(ConfigSet_in(input_files=fit_config_file),
-              ACE_name='ACE.B_test', params=params, ref_property_prefix='REF_',
-              run_dir=str(run_dir), skip_if_present=True)
+    ACE = prepare_params_and_fit(
+        ConfigSet_in(input_files=fit_config_file),
+        ACE_fname_stem='ACE.B_test', 
+        ace_fit_params=params, 
+        ref_property_prefix='REF_',
+        run_dir=str(run_dir), 
+        skip_if_present=True)
     time_actual = time.time() - t0
     print('ACE', ACE)
 
     assert os.path.exists(os.path.join(tmp_path, run_dir, f'ACE.B_test.json'))
-    assert os.path.exists(os.path.join(tmp_path, run_dir, f'ACE.B_test.yace'))
+    # assert os.path.exists(os.path.join(tmp_path, run_dir, f'ACE.B_test.yace'))
```

Comment:
I'd like some sort of support for yace, but I'm happy to decide that there should be a JSON -> anything else converter in ACE1, rather than coding different format/suffix logic into the workflow wrapper.  Just making a note about it.

Reply:
Would you usually need either .json or .yace or both? If the former, ACE1pack could be modified to save to the appropriate format based on the suffix, if the latter, I think a converter script is better than having an option or default to save both both.

Reply:
Let's just support only JSON here - Christoph says a conversion script is nearly trivial.

Reply:
ok!